app/dashboard/templates/dashboard/sidebar_search.html

```diff
@@ -153,6 +153,19 @@
       </div>
     </div>
 
+    <div class="col">
+      <div class="col-12 subheading">
+        Applicants
+      </div>
+      <div class="col-12 options">
+        <select class="w-100 js-select2" name="applicants" id="applicants_box">
+          <option value="0">0</option>
+          <option value="1-5">1-5</option>
+          <option value="ALL">ALL</option>
```

Comment:
can we setup the default to "all"  so people will still see all bounties by default. Will be hard for users to understand if we show only 0 applicants by default and some of them will feel this as a bug.

Reply:
@PixelantDesign what do you think about "all" as default value?

Reply:
I second @octavioamu !

Reply:
Let's set default to all. Thanks @rafalkowalski!

Reply:
done

Reply:
Hey guys! I just looked at some data and think it's best if we filter by 0 applicants as a default. I don't want to post the data here, but hit me up if you guys want to chat.

@rafalkowalski could we make the change to default to 0? Thank you!

Reply:
Ok i've changed to 0 value as a default